src/main/webapp/app/overview/course-lectures/exercise-unit/exercise-unit.component.ts

```diff
@@ -22,5 +22,5 @@ export class ExerciseUnitComponent {
     course: Course;
 
     @Input()
-    isPresentationMode: false;
+    isPresentationMode = false;
```

Comment:
```suggestion
    isPresentationMode: boolean = false;
```
It's not important though, and I also don't know if it is better.

Reply:
It's not necessary because assigning a default boolean value automatically defines the type.